kbase-extension/static/kbase/js/widgets/function_output/kbaseAttributeMapping.js

```diff
@@ -14,11 +15,10 @@ define([
     'datatables.net-buttons-print',
     'knhx',
     'widgetMaxWidthCorrection',
-], (KBWidget, $, Config, kbaseAuthenticatedWidget) => {
+], (KBWidget, $, Config, Workspace, Runtime) => {
```

Comment:
The tests around this suite were consistently failing. In digging through why, I found a few oddities that were worth addressing.

Mainly, it was a race condition for starting and running the tests, due to the use of `kbaseAuthenticatedWidget` as a parent here.  Also, because of the call to `renderAndLoad` during `init` and in the `loggedInCallback`, that was being called twice.